laos_climate_change/emdat_processing.py

```diff
@@ -0,0 +1,74 @@
+import pandas as pd
+import os
+from os.path import exists
+
+
+def process_emdat(repo_path=".../laos-climate-change"):
+    if not exists(repo_path + "/data"):
+        os.makedirs(repo_path + "/data")
+
+    if not exists(repo_path + "/data/emdat.xlsx"):
+        raise NotImplementedError()
```

Comment:
Add an informative message inside the `NotImplementedError`, like this:

```
raise NotImplementedError("No EM-DAT data was found at `/data/emdat.xlsx`. Please make an account at <url>, download the database, and place it in `/data/emdat.xlsx`")
```

Reply:
Done!